app/schemas/models/cinematic.schema.js

```diff
@@ -0,0 +1,151 @@
+  }),
+  enterOnStart: {
+    type: 'boolean',
+    title: 'Animate in?',
+    description: 'If true the character will animate in. Otherwise the character will start simply there.'
+  },
+  position: c.point2d({ title: 'Position', description: 'Where character is located.' })
+})
+
+const ShotSetup = c.object({
+  title: 'ShotSetup'
+}, {
+  cameraType: c.shortString({
+    title: 'Camera Type',
+    description: 'The shot type',
+    enum: ['right-close', 'left-close', 'dual'],
+    default: 'dual'
+  }),
+  rightThangType: CharacterSchema('Right Character'),
+  leftThangType: CharacterSchema('Left Character'),
+  backgroundArt: c.object({
+    title: 'Background Art',
+    description: 'The art in the background of this shot.'
+  }, {
+    type: c.shortString({ enum: ['slug', 'null'] }),
+    slug: c.shortString({
+      title: 'Background path',
+      description: 'Path to the background asset'
+    })
+  })
+  // TODO: music
+  // TODO: next - id
+})
+
+const DialogNode = c.object({
+  title: 'Dialog Node',
+  description: 'A node of a shot. Contains dialog instructions.',
+  required: ['dialogClear']
+}, {
+  speaker: c.shortString({ enum: ['left', 'right'], title: 'Speaker', description: 'Which character is speaking. Used to select speech bubble.' }),
+  text: { type: 'string', title: 'Text', description: 'html text. TODO: how to translate this and interpolate this???' },
+  textLocation: c.object({ title: 'Text Location', description: 'An {x, y} coordinate point.', format: 'point2d', required: ['x', 'y'] }, {
+    x: { title: 'x', description: 'The x coordinate.', type: 'number', 'default': 0 },
+    y: { title: 'y', description: 'The y coordinate.', type: 'number', 'default': 0 } }),
+  action: c.shortString({ title: 'Action', description: 'The action or animation to play on the speaker.' }),
+  triggers: c.object({
+    title: 'Triggers',
+    description: 'Events that can occur during the dialogue.'
+  }, {
+    changeBackground: c.object({
+      title: 'Change Background',
+      description: 'Change the background image of the cinematic',
+      required: ['art', 'triggerStart']
+    }, {
+      art: c.shortString({ title: 'Art', description: 'The background art path' }),
+      triggerStart: c.int({ title: 'Trigger Start(ms)', description: 'The number of milliseconds until the background changes.' })
+    }),
+    backgroundObject: c.object({
+      title: 'Background Object',
+      description: 'Add a background object after given duration',
+      required: ['art', 'triggerStart']
+    }, {
+      thangType: c.shortString({ title: 'Art', description: 'The background image ThangType slug.' }),
+      triggerStart: c.int({ title: 'Trigger Start(ms)', description: 'The number of milliseconds until background image art appears' })
+    }),
+    clearObjects: c.object({
+      title: 'Clear Object',
+      description: 'Clears the background objects from the screen after a given duration'
```

Comment:
Does it clear all background objects or a particular one?

Reply:
Spec  says there will only ever be one at a time. So just that one.